### ISSUE TITLE
训练过拟合问题

### ISSUE THREAD

**Issue by alittelboy** [Aiwiscal/MIT_Scheme#1]
你好，我使用你的源代码在自己的设备上运行，但是训练结果是过拟合的，请问如何解决？
Epoch 1/50
27003/27003 [==============================] - 30s - loss: 1.0729 - categorical_accuracy: 0.9168 - val_loss: 8.8669 - val_categorical_accuracy: 0.0540
Learning rate:  0.1
Epoch 2/50
27003/27003 [==============================] - 21s - loss: 0.4415 - categorical_accuracy: 0.9863 - val_loss: 7.0452 - val_categorical_accuracy: 0.1123
Learning rate:  0.1

......

Epoch 49/50
27003/27003 [==============================] - 21s - loss: 0.0089 - categorical_accuracy: 1.0000 - val_loss: 11.6176 - val_categorical_accuracy: 0.0822
Learning rate:  0.001
Epoch 50/50
27003/27003 [==============================] - 21s - loss: 0.0088 - categorical_accuracy: 1.0000 - val_loss: 11.6722 - val_categorical_accuracy: 0.0816
